packages/react-charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -145,8 +159,8 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
           </FocusZone>
           <FocusZone direction={FocusZoneDirection.horizontal}>
             <div>
-              <svg className={this._classNames.chart} aria-label={data?.chartTitle}>
-                <g>{bars[0]}</g>
+              <svg ref={this.barChartSvgRef} className={this._classNames.chart} aria-label={data?.chartTitle}>
+                {bars[0]}
```

Comment:
why removed the g tag?

Reply:
I think this didn't serve any purpose here, all the individual bars are enclosed with their `<g>` tag

```
<svg>
<g /> // bar1
<g /> // bar2
<g /> // bar3
</svg>
```

This is also the case with the multiStackedBarChart, which follows the same structure.

Reply:
Lets keep it. Unless we are sure it doesn't have any accessibility or styling purpose.